src/js/letters/components/ErrorableSelect.jsx

```diff
@@ -109,7 +109,10 @@ ErrorableSelect.propTypes = {
     ])).isRequired,
   required: PropTypes.bool,
   includeBlankOption: PropTypes.bool,
-  value: PropTypes.string.isRequired,
+  value: PropTypes.shape({
+    value: PropTypes.string,
+    dirty: PropTypes.bool
```

Comment:
Are we doing anything with `dirty` now? Do we know the select will be able to handle this new data structure? Above we're just passing `this.props.value`, if we're changing this to an object we may need to instead pass it `this.props.value.value`.